src/documentation/io/built-in.md

```diff
@@ -24,11 +24,14 @@ Consult the [Programming Guide I/O section]({{site.baseurl }}/documentation/prog
 <tr>
   <td>Java</td>
   <td>
-    <p><a href="https://github.com/apache/beam/tree/master/sdks/java/io/hadoop-file-system">Apache Hadoop File System</a></p>
+    <p>Beam Java supports HDFS, S3, GCS and local filesystems out of the
```

Comment:
By "out of the box", is that saying "you don't need to specify a dependency"? Could we simplify it to:
Apache HDFS, Amazon S3, Google Cloud Storage, and local filesystems (no dependency needed)

I think this whole table is overdue for an overhaul -- perhaps as one IO per line that checks off what languages support it and lists the required dependency if applicable. I'll put that on my list.

Reply:
Done (but without "no dependency needed": I'm not aware of anyone developing a filesystem support for Beam for which a dependency is needed)